firecracker/rootfs/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1.3-labs
+
+FROM debian:bullseye-slim AS rootfs-build
+
+WORKDIR /home/grapl
+
+RUN --mount=type=cache,target=/var/lib/apt/lists,sharing=locked,id=rootfs-apt <<EOF
+    apt-get update
+    apt-get install --yes --no-install-recommends \
+        fuse=2.9.9-5 \
+        fuseext2=0.4-1.2+b1
+EOF
+
+COPY docker_runtime docker_runtime
```

Comment:
Am I missing this file/directory? Does this need to be committed? Also I don't see this being used

Reply:
It's there:
https://github.com/grapl-security/grapl/pull/1498/files#diff-5ac450fc9b2ea12729c5e5f795d6bbce5ea457229e6e24ce3f129f741fa5c7e2

and it's where we write the script that's actually done at `docker run` time: create_rootfs_image